websites/A/AniLibria/presence.ts

```diff
@@ -0,0 +1,133 @@
+const presence = new Presence({ clientId: "1165759293576982578" });
+
+let cachedTime = 0;
+
+export function adjustTimeError(time: number, acceptableError: number): number {
+	if (Math.abs(time - cachedTime) > acceptableError) cachedTime = time;
+	return cachedTime;
+}
+
+presence.on("UpdateData", async () => {
+	const { pathname, href } = document.location,
+		ogTitle = document
+			.querySelector('meta[property="og:title"]')
+			?.getAttribute("content"),
+		video = document.querySelector("video"),
+		PATHS = {
```

Comment:
Consider placing this variable outside of the main loop for readability and performance